docs/jupyter/bokeh.ipynb

```diff
@@ -0,0 +1,106 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Demonstrating Bokeh Modin Interoperability"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import modin.pandas as pd\n",
+    "import pandas as pandas\n",
+    "from bokeh.plotting import figure\n",
+    "from bokeh.models import ColumnDataSource\n",
+    "from bokeh.io import show"
+   ]
```

Comment:
All comments above are applicable to all notebooks. Please check every notebook.